### PR TITLE
Transition to r5.1

### LINKED ISSUE #141: Transition notes: addition of R5.1
Create new Section 13.4 Transitioning from COUNTER R5 to R5.1. To include
- item as unit of reporting
- data types
- access types
- report headers
- SUSHI and JSON changes
- smaller aspects

### PR DESCRIPTION
**Pull request by tashalouizamc**
Added new section 13.4, transitioning to r5.1. Note that we also have a more comprehensive friendly guide on the transition

### CODE REVIEW

File: source/13-transitioning/04-transitioning-from-counter-r5-to-r51.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* For the same reason, R5.1 does not use the Section_Type attribute.
+* The list of Data_Types has been expanded. This means that conference proceedings, for example, can be more accurately classified. Report consumers are advised to use the Title Report to gain a full understanding of usage, rather than using Standard Views of the Title Report which are restricted to reporting on books and journals.
+* The definition and application of Access_Types has been clarified. The primary impact here is on the TR_B1, TR_J1 and TR_J4 Standard Views of the Title Report, which will show only Controlled content without Free_To_Read materials. Report consumers are advised to use the Title Report to gain a full understanding of usage.
+* Components, an aspect of the Item Report, are optional in R5.1. Report consumers should sum up item usage from R5 Item Reports offering Components to compare like-for-like with item usage in R5.1 Item Reports not offering Components.
```
Comment (berndoberknapp) [inline source/13-transitioning/04-transitioning-from-counter-r5-to-r51.rst]: Only the Total_Item metrics are reported on the component level, the Unique_Item metrics are reported on the item level and can be compared directly.

**Review by berndoberknapp — CHANGES_REQUESTED** [review]
Please see the inline comment.

Note that I've fixed the file name and corrected a plural.

File: source/13-transitioning/04-transitioning-from-counter-r5-to-r51.rst (hunk 1 above)
Comment (berndoberknapp) [reply source/13-transitioning/04-transitioning-from-counter-r5-to-r51.rst]: What I was trying to say is that there is no need to sum up anything, you could simply compare the Unique_Item metrics.

Comment (tashalouizamc) [reply source/13-transitioning/04-transitioning-from-counter-r5-to-r51.rst]: Understood, amended and committed

**Review by berndoberknapp — APPROVED** [review]
Note that I've changed "without Free_To_Read" to "without Open and Free_To_Read", since Open seemed to be missing.

**Pull request merged**